datahub-web-react/src/app/homeV3/HomePage.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import PersonalizationLoadingModal from '@app/homeV2/persona/PersonalizationLoadingModal';
+import HomePageContent from '@app/homeV3/HomePageContent';
+import Header from '@app/homeV3/header/Header';
+import { HomePageContainer, PageWrapper, StyledVectorBackground } from '@app/homeV3/styledComponents';
+import { SearchablePage } from '@app/searchV2/SearchablePage';
+
+export const HomePage = () => {
```

Comment:
alrighty this change is good i think to rename to homev3, just going to be messy with the existing PRs we have up